app/javascript/controllers/date_controller.js

```diff
@@ -0,0 +1,67 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["preselected"]
+
+  connect() {
+    if (!this.isDisabled(this.preselectedTarget)) {
+      if (this.isDraft(this.preselectedTarget)) {
+        this.element.classList.add("bg-danger", "text-light")
+      } else {
+        this.element.classList.add("bg-primary", "text-light")
+      }
+    }
+
+    if (this.preselectedTarget.getAttribute("draft") === "true") {
```

Comment:
existing drafts render in yellow.

Reply:
This seems redundant with the `disabled` property - couldn't you just check if the element is disabled or not here?

Reply:
possibly naming confusion happening here: `drafts` are the episodes that have already been scheduled and will appear in yellow on the calendar. not necessarily tied to or mutually exclusive from the `disabled` property which is _any date_ chosen, so that's the reasoning behind it being a different attribute.

Reply:
Ohhhh... you're right.  I misread that.  Disregard!